db/migrate/20230411155437_set_dataload_job_defaults.rb

```diff
@@ -0,0 +1,8 @@
+class SetDataloadJobDefaults < ActiveRecord::Migration[7.0]
+  def change
+    change_table :dataload_jobs, bulk: true do |t|
```

Comment:
This change is because the committee gem currently does not support null strings for api response/request validation.